src/main/java/org/codehaus/mojo/wagon/CopyMojo.java

```diff
@@ -60,6 +61,20 @@ public class CopyMojo
     @Parameter( property = "wagon.caseSensitive")
     private boolean caseSensitive = true;
 
+    /**
+     * Local directory to store downloaded artifacts.
+     */
+    @Parameter( property = "wagon.downloadDirectory")
+    private File downloadDirectory;
+
+
+    /**
+     * Download files that doesn't exist in local directory.
+     * Upload files that are not yet in target.
+     */
+    @Parameter( property = "wagon.incremental")
+    private boolean incremental;
```

Comment:
I am not sure we should call it "incremental". Maybe something like `continue`? This would be closer to tools like `curl`. However, not individual (partial) files are continued.

Reply:
the property has been renamed